src/ffi/mod.rs

```diff
@@ -831,6 +844,72 @@ extern "C" {
     fn fortify_source();
 }
 
+#[cfg(feature = "hide_ips")]
+struct LogCensor {
+    ip_mask_seed: [u8; 32],
+    ip_masks: parking_lot::Mutex<HashMap<Ipv4Addr, String>>,
+    ip_regex: Regex,
+}
+
+#[cfg(feature = "hide_ips")]
+impl LogCensor {
+    fn new() -> Option<Self> {
+        Regex::new(
+            r"(?x)\b
+                (?:
```

Comment:
This additional complexity exsists, to avoid capturing local ips ?

Reply:
No, to capture just IPs ;)